.env.example

```diff
@@ -0,0 +1,3 @@
+NODE_ENV=development
+IVLE_API_KEY=
+IVLE_API_TOKEN=
```

Comment:
Where do we store the credentials for these? Can add to our spreadsheet if it's not already there?